monitors/test/BalanceMonitor.js

```diff
@@ -86,7 +86,6 @@ contract("BalanceMonitor.js", function(accounts) {
   it("Correctly emits messages on balance threshold", async function() {
     // Update the client.
     await tokenBalanceClient.update();
-    await tokenBalanceClient.update();
```

Comment:
As token balances are returned immediately after a query, there is no longer a need to update the client two times.

Reply:
Thank god